contribute.html

```diff
@@ -0,0 +1,434 @@
+					<div class="col-md-4">
+						<a href="images/contribute/investigate.jpg" target="_blank" rel="noopener">
+							<img src="images/contribute/investigate.jpg" alt="Diagnose" loading="lazy">
+						</a>
+					</div>
+					<div class="col-md-8">
+						<div class="project-description">
+							<div class="project-title clearfix">
+								<h3>Investigate issues / Provide help</h3>
+								<span class="show_hide close"></span>
+							</div>
+							<p>Many times the DietPi project needs help on existent items:
+								<ul>
+									<li>Run an investigation (e.g. an installation issue on specific software title)</li>
+									<li>Provide an workaround to an existent issue</li>
+									<li>Give help when testing is required</li>
+								</ul>
+								<br/>Browse <a href="https://github.com/MichaIng/DietPi/issues" target="_blank" rel="noopener">DietPi issues on GitHub</a> or <a href="https://github.com/MichaIng/DietPi-Docs/issues" target="_blank" rel="noopener">DietPi documentation tasks on GitHub</a>.
+							</p>
+						</div>
+					</div>
+				</div>
+				<!-- End details for portfolio project 2 -->
+				<!-- Start details for portfolio project 3 -->
+				<div id="slidingdiv3" class="toggleDiv row single-project">
+					<div class="col-md-4">
+						<a href="images/contribute/question.jpg" target="_blank" rel="noopener">
+							<img src="images/contribute/question.jpg" alt="Question" loading="lazy"></a></div>
+					<div class="col-md-8">
+						<div class="project-description">
+							<div class="project-title clearfix">
+								<h3>Give help / Answer questions</h3>
+								<span class="show_hide close"></span>
+							</div>
+							<p><b>Core idea:</b> There are many questions on GitHub, and also on Forum. Maybe for some of the issues, we need more insights.</p>
+							<p>Sometimes there are open questions where you could know the answer. 
+								<br/>Or help is needed for a specific installation (e.g. a driver installation for a specific SBC, or a check of a performance tweak etc.)
+								<br/><br/>Browse <a href="https://github.com/MichaIng/DietPi/issues" target="_blank" rel="noopener">DietPi issues on GitHub</a> and find these items.
+							</p>
+						</div>
+					</div>
+				</div>
+				<!-- End details for portfolio project 3 -->
+				<!-- Start details for portfolio project 4 -->
+				<div id="slidingdiv4" class="toggleDiv row single-project">
+					<div class="col-md-4"><a href="images/contribute/test-configurations.jpg" target="_blank" rel="noopener">
+						<img src="images/contribute/test-configurations.jpg" alt="GNUstep" loading="lazy"></a></div>
+					<div class="col-md-8">
+						<div class="project-description">
+							<div class="project-title clearfix">
+								<h3>Test configurations</h3>
+								<span class="show_hide close"></span>
+							</div>
+							<p><b>Core idea:</b> Very similar with investigate issues, but more specific to hardware - either specific SBC models or PCs</p>
```

Comment:
`...specific SBC models or PCs</p>` -> `...specific SBC models or PCs.</p>`

Full stop at end of sentence is missing.